src/status_im/ui/screens/discover/dapp_details/views.cljs

```diff
@@ -27,14 +27,14 @@
      [react/view styles/dapp-details-container
       [react/view styles/dapp-details-header
        [react/view styles/dapp-details-icon
-        [chat-icon.screen/chat-icon photo-path {:size 50}]]
+        [chat-icon.screen/chat-icon photo-path {:size 56}]]
        [react/view styles/dapp-details-name-container
         [react/text {:font  :medium
                      :style styles/dapp-details-name-text} name]]]
       [react/view styles/dapp-details-action-container
        [react/view styles/dapp-details-action-icon-container
-        [vector-icons/icon :icons/open {:color :active
-                                        :style styles/dapp-details-open-icon}]]
+        [vector-icons/icon :icons/open {:color           :active
+                                        :style           styles/dapp-details-open-icon}]]
```

Comment:
Any specific reason for this change?

Reply:
Accidental. Had a more substantial change, undid it but not everything. It won't be a merge issue this time specifically, but I see your point.